packages/lesswrong/components/comments/CommentsItem/CommentPollVote.tsx

```diff
@@ -49,10 +57,15 @@ const CommentPollVote = ({ comment, classes }: { comment: CommentsList; classes:
   const showStartAgreement = startAgreement !== endAgreement;
   const showStartPercentage = showStartAgreement || endPercentage !== startPercentage;
 
+  const CurrentVoteTag = isGlobal ? 'span' : 'a';
+
+  const questionWording = forumEvent?.pollQuestion?.html && stripFootnotes(forumEvent.pollQuestion.html);
+
   return (
     <span className={classes.root}>
       {showStartPercentage && (
         <span className={classNames(startAgreement ? classes.agreePollVote : classes.disagreePollVote)}>
+          {/* TODO note that I would prefer to remove this */}
           <LWTooltip title="Vote when comment was posted" placement="top" popperClassName={classes.tooltip}>
             <s>
               {startPercentage}
```

Comment:
@agnesstenlund I think this is obvious and now there are two tooltips is a bit cluttery, can I remove it?